src/Microsoft.Tye.Core/ProcessUtil.cs

```diff
@@ -129,15 +150,21 @@ public static class ProcessUtil
                     process.WaitForExit(ProcessExitTimeoutMs);
```

Comment:
`process.WaitForExit()` supposed to wait for the output and error stream to complete, so, if that is the case, when we get to read from `outputBuilder` and `errorBuilder`, it should be completely written right?

Reply:
For `Process.WaitForExit()` that is the case, but not `Process.WaitForExit(int)`, which can timeout, in which case output may continue to be written.

Reply:
Ah. Just noticed your note/comment.

Reply:
I believe we switched to `Process.WaitForExit(int)` to avoid a deadlock when the process doesn't want to die (which happens in certain cases of a .NET process starting *another* .NET process and piping the output).